plugins/zynaddsubfx/ZynAddSubFx.cpp

```diff
@@ -507,19 +507,19 @@ ZynAddSubFxView::ZynAddSubFxView( Instrument * _instrument, QWidget * _parent )
 	m_portamento->setLabel( tr( "PORT" ) );
 
 	m_filterFreq = new Knob( knobBright_26, this );
-	m_filterFreq->setHintText( tr( "Filter Frequency:" ), "" );
+	m_filterFreq->setHintText( tr( "Filter frequency:" ), "" );
 	m_filterFreq->setLabel( tr( "FREQ" ) );
 
 	m_filterQ = new Knob( knobBright_26, this );
-	m_filterQ->setHintText( tr( "Filter Resonance:" ), "" );
+	m_filterQ->setHintText( tr( "Filter resonance:" ), "" );
```

Comment:
What's the rationale for writing "Q/Resonance" in some places (e.g. InstrumentSoundShaping) and plain old "resonance" in others?

Reply:
I forgot to change these, however, I'm not sure which term is more appropriate, so I'll revert these changes for now.